### PR TITLE
3djc/horus name on mixers

### PR DESCRIPTION
**Pull request by 3djc**
Improvement of model setup mixers view:
- real time mixer out and output view for selected mix on bottom bar (output name also displayed)
- use of icons for mix type
- optimisation of columns size and position to allow more mix settings configurations to display nicely

image

### CODE REVIEW

**Comment by bsongis** [comment]
Is the display OK when using GVARS?

**Comment by bsongis** [comment]
Perhaps the little += *= := should be shifted a little bit to the right

File: radio/src/gui/horus/bitmapbuffer.cpp [hunk 1]
```diff
@@ -507,6 +507,20 @@ BitmapBuffer * BitmapBuffer::loadMask(const char * filename)
   return bitmap;
 }
 
+
+BitmapBuffer * BitmapBuffer::loadAndConvertMask(const char * filename, int color, int bgcolor)
```
Comment (bsongis) [inline radio/src/gui/horus/bitmapbuffer.cpp]: I am almost sure I already added this function somewhere, perhaps in default.cpp (gui/horus/themes)

Comment (bsongis) [inline radio/src/gui/horus/bitmapbuffer.cpp]: But yes this is the right place

File: radio/src/gui/horus/menu_model_mixes.cpp [hunk 2]
```diff
@@ -482,11 +514,12 @@ bool menuModelMixAll(evt_t event)
         }
         else if (sub == cur) {
           s_currIdx = i;
+          displayMixStatus(ch - 1);
         }
         if (cur-menuVerticalOffset >= 0 && cur-menuVerticalOffset < NUM_BODY_LINES) {
           LcdFlags attr = ((s_copyMode || sub != cur) ? 0 : INVERS);
 
-          if (mixCnt > 0) lcdDrawTextAtIndex(6, y, STR_VMLTPX2, md->mltpx, 0);
+          if (mixCnt > 0) lcd->drawBitmap(6, y, mpx_mode[md->mltpx]);
```
Comment (bsongis) [inline radio/src/gui/horus/menu_model_mixes.cpp]: perhaps 10 instead of 6

File: radio/src/gui/horus/bitmapbuffer.cpp [hunk 3]
```diff
@@ -507,6 +507,20 @@ BitmapBuffer * BitmapBuffer::loadMask(const char * filename)
   return bitmap;
 }
 
+
+BitmapBuffer * BitmapBuffer::loadAndConvertMask(const char * filename, int color, int bgcolor)
+{
+  BitmapBuffer * source = BitmapBuffer::loadMask(filename);
+  static BitmapBuffer * dest;
+  
+  dest = new BitmapBuffer(BMP_RGB565, source->getWidth(), source->getHeight());
```
Comment (bsongis) [inline radio/src/gui/horus/bitmapbuffer.cpp]: SEGFAULT if the file is not present!

File: radio/src/gui/horus/bitmapbuffer.cpp [hunk 4]
```diff
@@ -507,6 +507,20 @@ BitmapBuffer * BitmapBuffer::loadMask(const char * filename)
   return bitmap;
 }
 
+
+BitmapBuffer * BitmapBuffer::loadAndConvertMask(const char * filename, int color, int bgcolor)
+{
+  BitmapBuffer * source = BitmapBuffer::loadMask(filename);
+  static BitmapBuffer * dest;
```
Comment (bsongis) [inline radio/src/gui/horus/bitmapbuffer.cpp]: Not static!

**Comment by 3djc** [comment]
All broken again, will fix it tomorow, I HATE git rebase

**Pull request closed without merging**